src/main/resources/defaultProperties/WebCapabilities.properties

```diff
@@ -10,6 +10,7 @@ mobileEmulation.width=
 mobileEmulation.height=
 mobileEmulation.pixelRatio=
 mobileEmulation.userAgent=
+baseURL=https://demoqa.com
```

Comment:
can we set the default to empty please? we also need to add some validations to ensure that "empty" doesn't throw any exceptions.

Reply:
<img width="1511" alt="Screenshot 2022-12-14 at 3 36 11 PM" src="https://user-images.githubusercontent.com/38945724/207610998-18db707f-ee8b-43f9-b4ad-4dde65043e39.png">

When a user selects to navigate with the new feature but didn't add a base Url , SHAFT will throw an error as described at the screenshot